.travis.yml

```diff
@@ -10,10 +16,39 @@ branches:
   only:
   - master
 
-before_script:
-- rustup component add rustfmt
+    env:
+      global:
+        - RUST_BACKTRACE=1
+        - secure: "gV/Zl5ihkZTB3S8ezmpTD7gvAzI2+tOSEQhZr91XoLptLt/w0y69q9ggOUiC457tD+77nb5SgqfRNI1PC+57xWpky1SowKJLzzmEbSdGdv+D35WcJVED4sasyEVPKolO3uOdW6h6vCOpRkyEOqHI4q/x7TUZu1ZiqrXHaU3U7s8MRceRCdzFRyC9l6RF9ovyIz/4aagfIvMPq5KB7WfmoJmkj/MDtNoUwuF8pry1HIbvPkx1Us7iYxVuzzUpuZOyp2dm1/7ERgTxURI9JwokvRxHW4EyBH3QWoKpMRIQZNQbjfluviSBKGpqXkBrsnwagDuVyC1tlUR2ZJOaJtAJcSlKaljLlj3pfPHYCORAFSKFyi3BKSNBf9kzVNcYVuZgosiXSaCzD1QYBlfBRcuDVgVPaxViMRmHAVZmYZBKOUO6YLasZIIkHzERAuhLF/vpetDIR/2A3cWbwknz55NXvvo5rY28a5YYEuactaJrTPbOP4y6WW22E2XkcuWcRN+TPN1bhoUctqT2DXxvlqqPTIILae/WV9hKFvB3bfnQVyB8BHldSmjLsLwQC4TzAJEIgegfC431bZGFn2pRTwJDtpiocThmzkWn8UOf9jCFTin4VZcQxEr6vZusKY77PObTEMQTJpiz6LSYDsTTMgTc5lmi8zJ0E/JlkL94vbL0SsM="
+        - secure: "p1XKGf0G74t6shD456Bbc89K83lLx8M5JZpik/7gIBwnninm5FVE+8Zm9+88gjIBSlcx9Rlp2P01va6fiTGDPTk3hDmioCJIifQSrCakd6/hiANu8IVZqj9ud83lBYuWYOY2kjTtFKV4e2JO80lny2+w6+cs3vGbltTxDwF0nAJV7T1kb2RMZOzUXd1/I0mAU68hg4EZ/tLV6H5g2I+Au1k0iST3EXxYaykGXntmDR7xv6QZboFT97ZUyb2gnqqM1MRui0aVft+XaFIy2/TQah2uwZy7cM+zzhBcVUSttiQqsNN8m789fo1xgZDFUR5tPNBBiztmRnk5/DvmESyGO8yFsy4mNFVplugyNpTq+7UBBxv2yf6CU1/TEtGNvbciTKt8celZajsm+o7Z+4kxUMOCLn1sS1wcoq3cnGHItSSHM72/ph3voJpe/4UxDMcnsWfQINLm13Q0ttNmx5hqwB9KinYor76UKE3LnfdT53G3x8/pdjkdQUURNB37A6S6UWWu7zZIJ1BgxQmQPBmIhlG98zVwqFtbuqMLX29rAUGR3Zu9YGmPcmFkoL9R2vfflUrUmdG4p5S/p9+F33PFpmSzb40I0ewlvgWyOddX/jo9/WRf0N943OxWjr7OEyIwC56t/q9zpwPo14qeu75S7L8DXNPyFONHiYddO+HhzBc="
+
+before_install:
+  # Check how much space we've got on this machine.
+  - df -h
+
+jobs:
+  include:
+    - stage: "Test and build"
+      name: "Run runtime tests"
+      script: RUST_TOOLCHAIN=nightly-2020-02-17 TARGET=runtime-test ./ci/script.sh
+    - # stage name not required, will continue last one
+      name: "Build (WASM and native)"
+      script: RUST_TOOLCHAIN=nightly-2020-02-17 TARGET=build-client ./ci/script.sh
+    - stage: "Build and push Docker image"
+      if: (NOT type IN (pull_request)) AND (branch = master)
+      script: travis_wait 180 ./ci/docker.sh
```

Comment:
I don't think we should publish a docker image of a library/module, but maybe I'm missing something – what's the reasoning behind that?